### DISCUSSION TITLE
Cannot use company-posframe-mode after upgrade

### DISCUSSION THREAD

**Discussion by thattemperature** [company-mode/company-mode#1483] [Q&A]
After I upgrade company-mode to the newest version in melpa(20240713.303), when I activate company-posframe-mode, It can pop up the box of candidates, but the box never disappear after I press return key (and any other keys), and the error message shown in echo area is: 
`company-call-frontends: Company: frontend company-pseudo-tooltip-unless-just-one-frontend error "Wrong type argument: stringp, nil" on command hide. `
The company-posframe-mode package did not update for a long time, so I think the upgrade of company-mode cause this problem.

**Comment by dgutov** [comment]
Thank you, moved to issues.
